src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -42,6 +42,20 @@
    [:f> zoomable-image/zoomable-image message index args]
    [rn/view {:style {:width constants/separator-width}}]])
 
+(defn on-scroll
+  [e item-width images-opacity landscape?]
+  (let [total-item-width (+ item-width constants/separator-width)
+        progress         (/ (if landscape?
+                              (oops/oget e "nativeEvent.contentOffset.y")
+                              (oops/oget e "nativeEvent.contentOffset.x"))
+                            total-item-width)
+        index-i          (max (Math/floor progress) 0)
```

Comment:
What's the meaning of `i` or `f` in `index-i`/`index-f`? Do you think we can come up with clearer names?

Reply:
@ilmotta it means `index-initial` and `index-final`. Renamed them to full names.

Reply:
Thank you :)